test/functional/test_site/expected/sub_site/testReuse._include_.html

```diff
@@ -2,14 +2,14 @@
 <p>Some variables:</p>
 <div></div>
 <p>IMG tags:
-  <img src="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png">
-  <img src="/test_site/sub_site/images/I'm not allowed to use my favorite tool.png">
+  <img src="images/I'm not allowed to use my favorite tool.png">
+  <img src="images/I'm not allowed to use my favorite tool.png">
```

Comment:
> * Tests still fail - testReuse.include.html doesn't have "/test_site/sub_site" anymore, but the output site looks correct. Still yet to determine whether this is an intended behaviour or the result of a bug.

Definitely a bug, this seems like a wrong change, because files that include this may not have the correct path references.

Also the output site doesn't look correct :P. I checked the test_site's `index.html`, the panels that include this HTML have broken images:

![broken](https://user-images.githubusercontent.com/3168908/56006436-627ca180-5d07-11e9-9faf-d373aa1db88b.png)